typescript/sdk/src/token/types.ts

```diff
@@ -166,3 +215,108 @@ function isCollateralRebasePairedCorrectly(
   );
   return allOthersSynthetic;
 }
+
+type CCIPContractExistsMap = ChainMap<ChainMap<boolean>>;
```

Comment:
Might be useful to have a comment on the intended population of the nested map, iiuc, its origin domain => destination domain => boolean exists